addons/comment-scroll/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Comment Scroll",
+  "description": "Makes Comments On the scratch website scrollable, which can reduce the height of the page.",
+  "credits": [
+    {
+      "name": "Genius398",
+      "link": "https://github.com/Genius398/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": [
+        "https://scratch.mit.edu/users/*",
+        "https://scratch.mit.edu/projects/*",
+        "https://scratch.mit.edu/studios/*"
+      ]
+    }
+  ],
+  "tags": ["comments", "community"],
+  "enabled_by_default": false
```

Comment:
Should be `enabledByDefault`.

e.g
https://github.com/ScratchAddons/ScratchAddons/blob/9afb7dc71684b42b3bc4ca8d4b6967ed9409849d/addons/remove-sprite-confirm/addon.json#L12

Reply:
I didn't write this, i'll fix though